EIPS/eip-4519.md

```diff
@@ -209,12 +209,12 @@ This EIP proposes using the Smart Contract to verify the mutual authentication p
 This EIP proposes the attributes timestamp (to register in the blockchain whenever the physical asset checks the tie with its token) and timeout (to register the maximum delay time established for the physical asset to prove again the tie). These attributes avoid a malicious owner or user could use the asset endlessly.
 
 **ERC-721-based**
-The [ERC-721](./eip-712.md) is a strong foundation for any NTF, nevertheless Ethereum addresses of assets and users, and authentication and key exchange processes are not considered. This EIP proposes an update of the [ERC-721](./eip-712.md) respecting backward compatibility.
+The [ERC-721](./eip-721.md) is a strong foundation for any NTF, nevertheless Ethereum addresses of assets and users, and authentication and key exchange processes are not considered. This EIP proposes an update of the [ERC-721](./eip-721.md) respecting backward compatibility.
   
 ## Backwards Compatibility
 This implementation is an extension of the ERC-721 standard, then it is not only compatible with the standard, but also is a way to improve some of the current NFT tokens based on the ERC-721 standard.
 ## Test Cases
-The test case presented in the paper shown below is addressed in **0x7eB5A03E7ED70ABf70fee48965D0411d37F335aC** and the code is available [here](../assets/eip-4519/PoC_SmartNFT/README.md)
+The test case presented in the paper shown below is addressed in **0x7eB5A03E7ED70ABf70fee48965D0411d37F335aC** in kovan testnet and the code is available [here](../assets/eip-4519/PoC_SmartNFT/README.md)
```

Comment:
```suggestion
The test cases presented in the paper shown below are available the assets folder [here](../assets/eip-4519/PoC_SmartNFT/README.md).
```

I would recommend removing the reference to the Kovan testnet address. Users should probably deploy their own test suites.